src/test/hammerdb/run_hammerdb.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# fail if trying to reference a variable that is not set.
+set -u
+# exit immediately if a command fails
+set -e
+# echo commands
+set -x
+
+rg=$1
+
+export RESOURCE_GROUP_NAME="${rg}"
+export AZURE_REGION=westus2
+# the branch name is stored in CIRCLE_BRANCH env variable in CI jobs.
+export BRANCH="${CIRCLE_BRANCH}"
+
+# add github to known hosts
+echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==" >> ~/.ssh/known_hosts
+
+set +x
+git config --global credential.helper store
+#TODO:: after hammerdbChanges is merged, use master.
```

Comment:
We can remove this and use master as the `hammerdbChanges` branch is merged to master.